image/postgres/rhtap.Dockerfile

```diff
@@ -37,11 +33,9 @@ RUN dnf upgrade -y --nobest && \
     rpm --verbose -e --nodeps $(rpm -qa curl '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*') && \
     rm -rf /var/cache/dnf /var/cache/yum
 
-COPY --from=builder --chown=postgres:postgres \
-    /stackrox/image/postgres/scripts/docker-entrypoint.sh \
-    /usr/local/bin/
-COPY --from=builder \
-    /stackrox/image/postgres/scripts/init-entrypoint.sh \
+USER 70:70
+
+COPY image/postgres/scripts \
     /usr/local/bin/
```

Comment:
I hate doing this, but I'd ask you to check with the Core Workflows team, probably @connorgorman and @c-du, why at all do we do `--chown=postgres:postgres` on `docker-entrypoint.sh` and don't do that on `init-entrypoint.sh` in the [downstream dockerfile](https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-central-db/Dockerfile.in?ref_type=heads#L41).

Even though it's a negligible risk, having scripts mutable by the user from whom the container executes gives a way to inject custom code in the container.

Note that after you moved `USER 70:70` above, `init-entrypoint.sh` will be owned by `postgres:postgres` unlike how it was owned by `root:root` before.

Reply:
Yes, I was aware of this change. Asked in Slack: https://redhat-internal.slack.com/archives/C01DQH7UP8C/p1699948813746169.

Reply:
TLDR of Slack conversation:

```
# docker run -it --entrypoint /bin/sh quay.io/redhat-user-workloads/rh-acs-tenant/acs/central-db:on-pr-de7e68e6a4e98b892fff8ce8148619d5ae32c67c
sh-4.4$ ls -lisa /usr/local/bin/
total 28
492156  4 drwxr-xr-x 1 root root  4096 Nov 15 09:44 .
492155  4 drwxr-xr-x 1 root root  4096 Oct 23 10:35 ..
491489 16 -rwxrwxr-x 1 root root 12349 Nov 15 09:33 docker-entrypoint.sh
491490  4 -rwxrwxr-x 1 root root   249 Nov 15 09:33 init-entrypoint.sh
sh-4.4$ docker-entrypoint.sh
Waiting for child process 16 to exit
flock: bad file descriptor: '/var/lib/postgresql/data/pglock'
sh-4.4$ init-entrypoint.sh
initdb: error: no data directory specified
You must identify the directory where the data for this database system
will reside.  Do this with either the invocation option -D or the
environment variable PGDATA.
sh-4.4$ whoami
postgres
sh-4.4$ id
uid=70(postgres) gid=70(postgres) groups=70(postgres)
```

This was my procedure to ensure the scripts can be executed by `postgres`, when owned by `root`. Also, the scripts are not world-writeable (`-rwxrwxr-x`), so if a malicious user came in as the postgres user, they won’t be able to modify the scripts.